bin/toolbox

```diff
@@ -399,7 +409,12 @@ main() {
             else
                 run "$@"
             fi
-            cleanup
+            #If nothing else is running, stop the container. Otherwise unset exit trap
+            if [ `${CLI} stats --no-stream --no-reset --format="{{.PIDS}}" ${TOOLBOX_NAME}` -lt 2 ]; then
+                cleanup
+            else
+                trap - EXIT
+            fi
```

Comment:
Please make this an extra PR with a description, which problem this solves and/or what the use-case is.

Reply:
Yes, this seems like something that could be interesting. Can you, as @thkukuk is asking, put this changes in it's own commit and explain a little bit more:
* in what cases this is/could be useful
* how cleaning up will work, in the various cases
* ?

Reply:
Sure thing.